utils/build.ps1

```diff
@@ -1008,27 +1032,47 @@ function Get-Dependencies {
     }
   }
 
-  function Install-PythonWheel([string] $ModuleName) {
+  function Is-Python-Module-Installed([string] $ModuleName) {
```

Comment:
Functions are not named using kebab case, there is a dictated nomenclature for PowerShell functions. https://learn.microsoft.com/en-us/powershell/scripting/learn/ps101/09-functions?view=powershell-7.5

Reply:
I did not know about that convention, fixed, thanks!